.goreleaser.yaml

```diff
@@ -18,6 +18,10 @@ builds:
       - darwin
     goarch:
       - amd64
+      - arm64
+    ignore:
+      - goos: windows
+        goarch: arm64
 archives:
   -
     name_template: "{{ .Binary }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}"
```

Comment:
Can you remove here the `{{ if .Arm }}v{{ .Arm }}` It's not necessary and it will break the hub functionality/convention.